### PR TITLE
Added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by settlersxp**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Runner Manager Contribution and Governance Policies
+This document describes the contribution process and governance policies of the Runner Manager project. The project is also governed by QAlity tech.
```
Comment (rolandleth) [inline CONTRIBUTING.md:2]: Qality Tech

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# Contribution Process
+Before making a contribution, please take the following steps:
+
+- Check whether there's already an open issue related to your proposed contribution. If there is, join the discussion and propose your contribution there.
+- If there isn't already a relevant issue, create one, describing your contribution and the problem you're trying to solve.
+- Respond to any questions or suggestions raised in the issue by other developers.
+- Fork the project repository and prepare your proposed contribution.
+- Submit a pull request.
+
+NOTE: All contributors must have a contributor license agreement (CLA) on file with before their pull requests will be merged. Please review the contribution requirements and submit (or have your employer submit) the required CLA before submitting a pull request.
```
Comment (rolandleth) [inline CONTRIBUTING.md:13]: on file with us?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+# Governance
+## Roles
+The project community consists of Contributors and Maintainers:
+
+A Contributor is anyone who submits a contribution to the project. (Contributions may include code, issues, comments, documentation, media, or any combination of the above.)
+A Maintainer is a Contributor who, by virtue of their contribution history, has been given write access to project repositories and may merge approved contributions.
+The Lead Maintainer is the project's interface with the QAlity Tech team. They are responsible for approving quarterly project reports and communicating on behalf of the project. The Lead Maintainer is elected by a vote of the Maintainers.
```
Comment (rolandleth) [inline CONTRIBUTING.md:21]: Maybe add an owner role, to describe us?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Contribution Rules
+Anyone is welcome to submit a contribution to the project. The rules below apply to all contributions. (The key words "MUST", "SHALL", "SHOULD", "MAY", etc. in this document are to be interpreted as described in IETF RFC 2119.)
+
+- All contributions MUST be submitted as pull requests, including contributions by Maintainers.
+- All pull requests SHOULD be reviewed by a Maintainer (other than the Contributor) before being merged.
+- Pull requests for non-trivial contributions SHOULD remain open for a review period sufficient to give all Maintainers a sufficient opportunity to review and comment on them.
+- After the review period, if no Maintainer has an objection to the pull request, any Maintainer MAY merge it.
```
Comment (rolandleth) [inline CONTRIBUTING.md:29]: Maybe only let leads merge?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+All Maintainer votes MUST be carried out transparently, with all discussion and voting occurring in public, either:
+- in comments associated with the relevant issue or pull request, if applicable;
+- on the project mailing list or other official public communication channel; or
+- during a regular, minuted project meeting.
+
+## Maintainer Qualifications
+Any Contributor who has made a substantial contribution to the project MAY apply (or be nominated) to become a Maintainer. The existing Maintainers SHALL decide whether to approve the nomination according to the Maintainer Voting process above.
+
+# Changes to this Document
+This document MAY be amended by a vote of the Maintainers according to the Maintainer Voting process above.
```
Comment (rolandleth) [inline CONTRIBUTING.md:50]: I would replace all all-caps words with normal caps

File: CONTRIBUTING.md (hunk 3 above)
Comment (rolandleth) [inline CONTRIBUTING.md:21]: Qality

**Review by rolandleth — CHANGES_REQUESTED** [review]

**Comment by rolandleth** [comment]
Do we really want such a complicated contribution? Do people really take time to vote and micromanage open source repos?

File: CONTRIBUTING.md (hunk 2 above)
Comment (settlersxp) [reply CONTRIBUTING.md:13]: according to https://en.wikipedia.org/wiki/Contributor_License_Agreement it is not a bad idea. However, we can remove that line and go for https://docs.fedoraproject.org/en-US/legal/fpca/ as the WIKI suggested.

**Comment by settlersxp** [comment]
> Do we really want such a complicated contribution? Do people really take time to vote and micromanage open source repos?

If they don't we will merge as owners after a period of time. If some do they have a framework to do it. We will read their contributions anyway so it doesn't have an impact at the moment. Maybe it will serve us in the future.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rolandleth) [reply CONTRIBUTING.md:13]: Huh? I meant there's a typo, it should be "on file with us", it's currently "on file with" 😄

File: CONTRIBUTING.md (hunk 3 above)
Comment (rolandleth) [reply CONTRIBUTING.md:21]: Still a typo

**Review by rolandleth — CHANGES_REQUESTED** [review]
.

**Review by rolandleth — APPROVED** [review]

**Pull request merged**